src/js/components/graph/graph.jsx

```diff
@@ -57,7 +57,11 @@ let Graph = React.createClass({
   },
 
   componentDidMount() {
-    this.graph = new GraphD3(this.getGraphEl(), 'main')
+    let showWelcome = !localStorage.getItem('jolocom.welcome')
```

Comment:
Shouldn't handle localStorage directly in components, move this to a store (accounts for example)